packages/amplication-client/src/Resource/create-resource/CreateServiceWizard.tsx

```diff
@@ -402,6 +402,7 @@ const CreateServiceWizard: React.FC<Props> = ({
             isOverrideGitRepository: isOverrideGitRepository,
           };
         }
+        const requireAuthenticationEntity = authType === "core";
```

Comment:
Is this true if and only if the user chooses to create the service with auth? (just checking with you since I don't know the wizard code well)

Reply:
@mulygottlieb  yes